src/containers/BookForm.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { createBook } from '../actions'
+import styled from '@emotion/styled'
+
+const Template = styled.div`
+  margin-bottom: 40px;
+  h2 {
+    font-weight: bold;
+    font-size: 20px;
+    color: #888;
+    margin-top: 40px;
+    padding-top: 29px;
+    border-top: 1px solid #e8e8e8;
+    margin-bottom: 19px;
+  }
+  form {
+    display: flex;
+  }
+  select {
+    width: 285px;
+    margin: 0 34px;
+  }
+  input {
+    flex: 1;
+  }
+`
+
+const SubmitButton = styled.button`
+  background-color: #0290ff;
+  color: white;
+  border: none;
+  height: 45px;
+  width: 184px;
+  border-radius: 3px;
+  font-size: 13px;
+  cursor: pointer;
+`
+
+const CATEGORIES = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'] 
+
+class BookForm extends React.Component {
+  state = {
+    input: '',
+    category: CATEGORIES[0]
+  }
+
+  handleTitleChange = (e) => {
+    this.setState({
+      input: e.target.value
+    })
+  }
+
+  handleCategoryChange = (e) => {
+    this.setState({
+      category: e.target.value
+    })
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault()
+    if (this.state.input === '') return alert('Please add a book name')
```

Comment:
Instead of using `alert` let's use a local error state. Or better yet, since in the backend your `book` model will have a validation for the attributes, then we can dispatch the specific error message from the backend if a user doesn't fill up all the fields and tries to submit the form. Then we can get that error state from the store and conditionally display it here.